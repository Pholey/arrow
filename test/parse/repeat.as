// Statement (may not be an expression)
while condition {
  break;
  continue;
}
