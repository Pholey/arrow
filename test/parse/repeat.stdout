{
  "tag": "Module",
  "span": "test/parse/repeat.as:1-5",
  "statements": [
    {
      "tag": "Repeat",
      "span": "test/parse/repeat.as:2-5",
      "condition": {
        "tag": "Identifier",
        "span": "test/parse/repeat.as:2:7-16",
        "text": "condition"
      },
      "statements": [
        {
          "tag": "Break",
          "span": "test/parse/repeat.as:3:3-8"
        },
        {
          "tag": "Continue",
          "span": "test/parse/repeat.as:4:3-11"
        }
      ]
    }
  ]
}
